frontend/src/app/main/ui/workspace/colorpicker/ramp.cljs

```diff
@@ -63,38 +64,52 @@
            :h h :s s :v v
            :r r :g g :b b)))
 
-(mf/defc ramp-selector
+(mf/defc ramp-selector*
   [{:keys [color disable-opacity on-change on-start-drag on-finish-drag]}]
-  (let [internal-color
+  (let [internal-color*
         (mf/use-state #(enrich-color-map color))
 
-        {:keys [h s v hex alpha]} @internal-color
+        internal-color
+        (deref internal-color*)
```

Comment:
Storing the state value in a variable avoids `deref`ing it each time it's used.

`deref` and `@` are equivalent, this is a matter of coherent style.